designsystem/src/main/java/com/babylon/wallet/android/designsystem/composable/RadixPrimaryButton.kt

```diff
@@ -72,7 +73,7 @@ fun RadixPrimaryButton(
                 )
             } else {
                 icon?.invoke()
-                Text(text = text, style = RadixTheme.typography.button)
+                Text(text = text, style = RadixTheme.typography.button.copy(fontSize = 16.sp))
```

Comment:
Maybe update the `fontSize` in `RadixTypography` directly, because as I can see it's always set to 16.sp.